Services/Administration/OJS.Services.Administration.Business/Problems/ProblemsBusinessService.cs

```diff
@@ -299,12 +300,13 @@ private async Task CopyProblemToContest(Problem? problem, int contestId, int? pr
         }
         else
         {
-            orderBy = await this.problemsData.GetNewOrderByContest(contestId);
+            orderBy = GlobalConstants.ProblemDefaultOrderBy;
```

Comment:
The `Problem` will always have the same `orderBy`?

Reply:
Fixed